src/ngraph/runtime/gpu/cuda_emitter.cpp

```diff
@@ -22,11 +22,39 @@
 #include "ngraph/runtime/gpu/cuda_emitter.hpp"
 #include "ngraph/runtime/gpu/gpu_primitive_emitter.hpp"
 #include "ngraph/runtime/gpu/gpu_runtime_context.hpp"
+#include "ngraph/runtime/gpu/gpu_util.hpp"
 #include "ngraph/runtime/gpu/type_info.hpp"
 #include "ngraph/util.hpp"
 
 using namespace ngraph;
 
+inline namespace enums
+{
+    enum pooling_indices
```

Comment:
I think you can replace this enum and vector<int> with a class that has int members N_, C_, ...
This should end up being more clear.

Reply:
Replaced with a struct and an ostream operator overload for printing it.